src/utilities/exitHandling.spec.ts

```diff
@@ -13,7 +13,9 @@ beforeEach(() => {
 });
 
 const mockProcessOn = mockSpy(jest.spyOn(process, 'on'));
-const mockProcessExit = mockSpy(jest.spyOn(process, 'exit'));
+const mockProcessExit = mockSpy(jest.spyOn(process, 'exit'), () => {
+  // Do nothing
```

Comment:
Found the fix here: https://stackoverflow.com/a/51448240/129437

Reply:
I would be better if we omit comments, default eslint configuration is forbidding those as well [no-inline-comments]

Reply:
If I omit the comment, that'd violate another rule:

![Screenshot_20230221_150928](https://user-images.githubusercontent.com/369863/220382808-86dffb3d-ab7c-4d35-b7b3-c57ebfc893fa.png)

Note that this isn't an inline comment though.

Reply:
approved :)

Reply:
Thanks! Will merge